app/controllers/transactions_controller.rb

```diff
@@ -7,6 +7,12 @@ def create
     @transaction.save
   end
 
+  def create_transfer
+    @transfer = Transfer.new(transaction_params)
+    @transaction = Transaction.new(transaction_params)
```

Comment:
@Gataniel @AlexanderUlitin yes, you don't need transaction here. Use transactions (if you need) only through transfer.